monero-outreach-docs/translations/archive/wallet_primer_fr.md

```diff
@@ -0,0 +1,90 @@
+Pour un niveau de sécurité équivalent à un véhicule blindé, vous devez utiliser un ordinateur qui ne sera plus jamais connecté à internet, et vous devez effacer sa mémoire après la génération de clés (de tel sorte que quelqu’un qui arrive à accéder physiquement à votre ordinateur ne peut pas les recréer). Presque tout le monde a un vieil ordinateur. Vous pouvez utiliser cela, ou acheter un nouvel ordinateur portable bas de gamme pour quelques centaines de dollars. Si vous imprimez vos clés de portefeuille à des fins de stockage, vous devez utiliser une imprimante sans mémoire intégrée (tel qu’une HP Deskjet 1100).  Vous devez vérifier que le hachage du logiciel tournant sur l’ordinateur servant de portefeuille (importé grâce à un CD ou une clé de stockage, car –encore une fois—il n’y a pas de connexion internet direct). Vous pouvez voir quelques exemples de tels hachages, pour le portefeuille officiel de Monero, à [getmonero.org/downloads](https://web.getmonero.org/downloads/). Quand le logiciel est signé, vous devriez vérifier la signature.
+
+### _Créer votre portefeuille_
+
+Avec les techniques et précautions décrites dans les sections précédentes, vous êtes maintenant prêt avec des clés générées de manière sûre, et tout ce qu’il reste à faire est de les stocker sur un portefeuille physique. L’approche la plus commune est d’imprimer la clé de dépense privée (sous une de ces trois formes) sur un papier. C’est ce que l’on appelle un portefeuille papier. Sauvegarder la clé d’audit privée et la clé publique de manière digital, où vous le voulez. 
+Ensuite vous pouvez cacher le portefeuille papier et envoyer des Monero dessus à votre convenance en utilisant la clé publique, qui est partageable. Et vous pouvez effectuer une vérification pour être sûre que les fonds arrivent en utilisant la clé d’audit privée.
+Vous pouvez aussi utiliser des méthodes qui ne sont pas basée sur du papier. La nomination portefeuille papier vient du fait que la plupart sont sous forme papier, mais pour cet article, la définition du portefeuille papier est étendue pour inclure d’autres mécanismes physiques d’enregistrement. Des outils sont commercialement disponibles afin de fabriquer des portefeuilles papiers en métal compatible avec Monero. Certains de ceux-ci glissent les symboles à leur emplacement, certain les gravent, et d’autres les emboutissent. Beaucoup marchent avec un standard, tel que le BIP39 (représentation 2, voir au-dessus). Afin de trouver des exemples, cherchez sur Amazon les termes « crypto-monnaies, acier, portefeuille » (bien que certains sont en fait faits en cuivre). Un bénéfice du portefeuille papier en métal est qu’il n’est pas tout aussi sensible à l’eau et au feu comparé au papier – Ceci aide à éviter de perdre le portefeuille, est se trouve probablement être le plus pertinent pour un niveau de sécurité équivalent à un véhicule blindé dont on a discuté plus haut.
```

Comment:
1. [ne sont pas basée ] ==> [ne sont pas **basées**  ; accord ]
2. [en fait faits en cuivre] ==> [en fait **fabriqués** en cuivre ; clareté, syntaxe ]
3. [il n’est pas tout aussi sensible à l’eau et au feu comparé au papier] ==> [il n'est pas **aussi sensible à l'eau et au feu que le papier** ; syntaxe, clareté]

Reply:
OK. For 3, just a question of taste for the sentence... but why not.